packages/preview/light-cv/0.1.0/typst.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "light-cv"
+version = "0.1.0"
+entrypoint = "template.typ"
+authors = ["Ansgar Lichter"]
+license = "MIT"
+description = "Minimalistic CV template for your own CV. Please install the font awesome fonts on your system before using the template."
+repository = "https://github.com/AnsgarLichter/cv-typst-template"
+keywords = ["cv", "curriculum vitae", "computer science", "application", "resume"]
+categories = ["cv"]
+disciplines = ["engineering", "computer-science"]
```

Comment:
Can this CV be used by professionals in other disciplines? If so, remove the disciplines key.